build-tools/libzip/libzip.props

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_LibZip>(MSBuildThisFileDirectory)../../../../../libzip</_LibZip>
```

Comment:
This should follow the convention of e.g. `$(JavaInteropSourceDirectory)` and `$(JavaInteropFullPath)` within `Configuration.props` and `Configuration.Override.props`, to allow the `libzip` directory to be overridden.

Thus, there should be new `$(LibZipSourceDirectory)` and `$(LibZipFullPath)` MSBuild properties, declared in `Configuration.props`.